cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/runtime/KubeTwillController.java

```diff
@@ -69,13 +69,15 @@
   private final String kubeNamespace;
   private final RunId runId;
   private final CompletableFuture<KubeTwillController> completion;
-  private boolean isStopped;
   private final DiscoveryServiceClient discoveryServiceClient;
   private final ApiClient apiClient;
   private final BatchV1Api batchV1Api;
   private final Type resourceType;
   private final V1ObjectMeta meta;
 
+  private volatile boolean isStopped;
```

Comment:
Looks like we are not using this field. Do we need it?

Reply:
Its still being used in line 540